### ISSUE TITLE
Error: unable to start blitz without `@prisma/cli` dependency

### ISSUE THREAD

**Issue by sirmyron** [blitz-js/blitz#1062]
### What is the problem?
If you don't have the `@prisma/cli` dependencies in your project, you are unable to run `blitz start`

### Steps to Reproduce

1. Remove `@prisma/cli` from package.json
2. `rm -r node_modules`
3. `blitz start`

and then you get this in the console:
```
node:19067) UnhandledPromiseRejectionWarning: Error: Oops, we can't find Prisma Client. Please make sure it's installed in your project
    at /development/blitz-app/node_modules/@blitzjs/cli/lib/src/commands/db.js:25:19
    at Generator.throw (<anonymous>)
    at rejected (/development/blitz-app/node_modules/tslib/tslib.js:113:69)
(Use `node --trace-warnings ...` to show where the warning was created)
(node:19067) UnhandledPromiseRejectionWarning: Unhandled promise rejection. This error originated either by throwing inside of an async function without a catch block, or by rejecting a promise which was not handled with .catch(). To terminate the node process on unhandled promise rejection, use the CLI flag `--unhandled-rejections=strict` (see https://nodejs.org/api/cli.html#cli_unhandled_rejections_mode). (rejection id: 2)
(node:19067) [DEP0018] DeprecationWarning: Unhandled promise rejections are deprecated. In the future, promise rejections that are not handled will terminate the Node.js process with a non-zero exit code.
Error: Oops, we can't find Prisma Client. Please make sure it's installed in your project
    at /development/blitz-app/node_modules/@blitzjs/cli/lib/src/commands/db.js:25:19
    at Generator.throw (<anonymous>)
    at rejected (/development/blitz-app/node_modules/tslib/tslib.js:113:69)
```

### Versions
`0.21.1`

**Comment by flybayer** [comment]
Oops, that shouldn't happen!

**Comment by enricoschaaf** [comment]
I would like to work on this but I dont know that the expected output should be. It seems like blitz start calls blitz db.runPrismaGeneration under the hood: 
```typescript
if (flags.production) {
  await prod(config, require("./db").runPrismaGeneration({silent: true, failSilently: true}))
} else {
  await dev(config, require("./db").runPrismaGeneration({silent: true, failSilently: true}))
}
```
The function need the @prisma/cli, in my mind we would have to provide a way to not run the db command on prisma start. Does this make sense?

**Comment by flybayer** [comment]
So I just removed that runPrismaGeneration stuff in https://github.com/blitz-js/blitz/pull/1121. Probably that fixes this issue!

Need to test for sure though

**Comment by enricoschaaf** [comment]
Just read the PR and seems like good stuff, from my understanding it should also fix this issue.

**Comment by flybayer** [comment]
Fixed by #1121
